cmd/veneur-emit/main.go

```diff
@@ -56,9 +64,23 @@ func main() {
 	if err != nil {
 		logrus.WithError(err).Fatal("Error!")
 	}
+	_ = conn
+
+	var nconn net.Conn
+	nconn, err = net.Dial("udp", addr)
+	if err != nil {
+		logrus.WithError(err).Fatal("Error!")
+	}
 
-	tags := tags(*tag)
-	err = sendMetrics(conn, passedFlags, *name, tags)
+	// tags := tags(*tag)
+
+	var span *ssf.SSFSpan
```

Comment:
Yeah, I think having a flag makes more sense - by default, emit StatsD, and you can pass a flag to change it to SSF.